crates/index-scheduler/Cargo.toml

```
@@ -22,13 +22,15 @@ flate2 = "1.0.30"
 meilisearch-auth = { path = "../meilisearch-auth" }
 meilisearch-types = { path = "../meilisearch-types" }
 page_size = "0.6.0"
+raw-collections = { git = "https://github.com/dureuill/raw-collections.git", version = "0.1.0" }
```

Should we change the URL here?

```suggestion
raw-collections = { git = "https://github.com/meilisearch/raw-collections.git", version = "0.1.0" }
```

---

Yes, we should. We should also release it on crates.io like the other crates ([in the _During Prelease_ section](https://github.com/meilisearch/meilisearch/issues/4985)).